### PR TITLE
Attn Ref Updates

### PR DESCRIPTION
**Pull request by cameron-eyds**
*Issue #:* /bcgov/entity#15916

*Description of changes:*
* Implements Staff implementation of Attn Ref component.
* Resource and config for attn ref
* Fixed payload for MHR Attn Ref field and Draft
* Misc table ux

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the PPR license (Apache 2.0).

### CODE REVIEW

File: ppr-ui/src/composables/mhrRegistration/useMhrValidations.ts [hunk 1]
```diff
@@ -38,6 +38,9 @@ export const useMhrValidations = (validationState: any) => {
     // Reset home location validations
     setValidation(MhrSectVal.LOCATION_VALID, MhrCompVal.LOCATION_TYPE_VALID, false)
     setValidation(MhrSectVal.LOCATION_VALID, MhrCompVal.CIVIC_ADDRESS_VALID, false)
+
+    // Reset Review validations
+    setValidation(MhrSectVal.REVIEW_CONFIRM_VALID, MhrCompVal.VALIDATE_STEPS, false)
```
Comment (cameron-eyds) [inline ppr-ui/src/composables/mhrRegistration/useMhrValidations.ts:43]: Fixes red border error when opening a transfer following a registration

File: ppr-ui/src/components/tables/common/TableRow.vue [hunk 2]
```diff
@@ -43,7 +43,7 @@
           <!-- child drafts will sometimes show outside their base reg during the sort -->
           <div
             v-if="isChild || (isDraft(item) && item.baseRegistrationNumber)"
-            :class="(isChild && !isTransAffi(item.registrationType))
+            :class="!(isChild && hasFrozenParentReg(item) && (isTransAffi(item.registrationType) || isDraft(item)))
```
Comment (cameron-eyds) [inline ppr-ui/src/components/tables/common/TableRow.vue:46]: Fixes alignment in Table for Children when parent has frozen states

File: ppr-ui/src/composables/mhrRegistration/useNewMhrRegistration.ts [hunk 3]
```diff
@@ -87,7 +87,7 @@ export const useNewMhrRegistration = () => {
         phoneExtension: ''
       },
       ownerGroups: [],
-      attentionReferenceNum: '',
+      attentionReference: '',
```
Comment (cameron-eyds) [inline ppr-ui/src/composables/mhrRegistration/useNewMhrRegistration.ts:90]: Referenced the schema and found this was being passed incorrectly 👍 
Fixed!

File: ppr-ui/src/components/mhrRegistration/ReviewConfirm/SubmittingPartyReview.vue [hunk 4]
```diff
@@ -91,7 +91,7 @@
 
           <v-row no-gutters class="px-6 py-7">
             <v-col cols="3">
-              <p class="side-header">Attention or<br>Reference Number</p>
+              <p class="side-header">Attention <span v-if="!isRoleStaffReg"> or<br>Reference Number</span></p>
```
Comment (dimak1) [inline ppr-ui/src/components/mhrRegistration/ReviewConfirm/SubmittingPartyReview.vue]: Can we use a `title` from a newly `attnRefConfig` here?

**Review by dimak1 — APPROVED** [review]
Nice use of the config 👏🏻

File: ppr-ui/src/composables/mhrRegistration/useNewMhrRegistration.ts (hunk 3 above)
Comment (doug-lovett) [reply ppr-ui/src/composables/mhrRegistration/useNewMhrRegistration.ts:90]: Nice catch.

**Comment by pwei1018** [comment]
Temporary Url for review: https://bcregistry-assets-dev--pr-1308-8k15st59.web.app

**Review by doug-lovett — APPROVED** [review]
Looks good.

File: ppr-ui/src/components/mhrRegistration/ReviewConfirm/SubmittingPartyReview.vue (hunk 4 above)
Comment (cameron-eyds) [reply ppr-ui/src/components/mhrRegistration/ReviewConfirm/SubmittingPartyReview.vue]: Yes! 
Definitely, good eye. Will fix

**Comment by pwei1018** [comment]
Temporary Url for review: https://bcregistry-assets-dev--pr-1308-8k15st59.web.app

**Pull request merged**